--- old
+++ new
@@ -47,0 +47,58 @@
+
+----
+[twitter](https://twitter.com/indiohistorian/status/1632185657031536640?t=iqpVjpSDjcpZMgjmFtq9pg&s=09 )
+
+----
+I've helped hundreds of people land 6-figure salaries. These 5 job interview phrases got them hired 'on the spot'
+https://artifact.news/s/NURpt7Yiz38=
+
+----
+PruForce & PruOne downtime: March 10, 12 AM to 5 AM
+
+Update to EQuotes system: March 10
+
+---
+# How to get people to trust you? (My intuition)
+- You are well informed. Shares informative and up-to-date news and insights.
+- You are the expert. You make educational content on your field of expertise.
+- You have a professional looking profile.
+- You are well-driven and motivated. You are a positive light. You share happiness and positivity.
+- You are approachable. You communicate on the  simple language and invites openness.
+- You are relatable. You do things people also do. You also think like your dream audience think.
+
+What to do:
+- Share at least one important thing/news daily.
+- Make educational content/story-driven content.
+    - _When to post and how many?_
+- Make a professional looking profile
+
+
+Posts
+1. Promotional content
+2. Educational content
+3. Story driven content
+4. Personal/Lifestyle -> boosts approachability 
+5. Motivational content
+
+
+----
+Wake up at 4 AM
+Prep (4 AM - 6 AM) - Breakfast
+Thesis (6 AM - 10 AM)
+
+
+
+
+Sleep at 10 PM
+
+---
+Best time to study: 6-10 AM
+
+
+Best time to post: 6 AM
+
+Best time to cold call: 10-11 AM, 12-1 PM, 7-8 PM
+Best time to engage in post/stories: 6-10 PM
+
+
+